potemkin/cloudformationstack.py

```diff
@@ -79,6 +81,9 @@ def _stack_outputs(self, stack_dict):
         """ Given the response from DescribeStacks, transform the outputs into a dictionary
 
         :param stack_dict: response from DescribeStacks """
+        if stack_dict["StackStatus"] != "CREATE_COMPLETE":
+            print(f'Stack creation error: {stack_dict["StackStatusReason"]}')
+            sys.exit()
```

Comment:
is there a good justification for calling exit() here vs. just throwing an exception?  doesn't matter _that_ much, but usually stay away from calling exit() unless i'm at the top level command line script.  hard to write automated tests for code that calls exit.

Reply:
Fixed

Reply:
fixed